### ISSUE TITLE
multiplier issues

### ISSUE THREAD

**Issue by Glanzer-modder** [tr4wzified/SpellTomePricePatcher#3]
I changed the multiplier in the settings.json file to 0.5 AFTER loading Synthesis (because every time I load Synthesis it OVERWRITES that file with the one from the repository!), but it still reports that the multiplier is 1 and no books get written to the Synthesis.esp plugin. How do I make the value persistent and take effect?

[154.1][SpellTomePricePatcher] *** DETECTED SETTINGS ***
[154.1][SpellTomePricePatcher] value_multiplier: 1

**Comment by Glanzer-modder** [comment]
Ok I kind of figured it out. I converted the whole thing to a local instance, then was able to see an error about converting a double (0.5) to a float so I had to add the 'F' suffix. I hardcoded the value in the Program.cs file and it works and all the book entries are now in Synthesis.esp:

float valueMultiplier = 0.5F;

**Comment by hoskonen** [comment]
Yeah, i can't see the difference in the prices. Tried with 3 or 3.0.

**Comment by Glanzer-modder** [comment]
hoskonen, did you create a local instance of this patcher? If not, every time you run the patcher it will download a fresh copy of the program.cs file and overwrite your changes.
